.github/workflows/tests.yml

```diff
@@ -28,6 +28,14 @@ jobs:
       uses: actions/setup-node@v2
       with:
         node-version: 16
+    - name: Determine deployer image
+      env:
+        PR_BODY: ${{ github.event.pull_request.body }}
+      run: |
+        export PR_DEPLOYER_IMAGE=$(echo $PR_BODY | grep -oP "deployer_image: \K\S+")
```

Comment:
cute, can we document this and the deploy-test label in the readme?

Reply:
Sure. I'll get it in followup just because I'm going to be redoing the readme anyway.